### PR TITLE
Shortest common supersequence

### PR DESCRIPTION
**Pull request by Amit366**
Added the code for Shortest common supersequence.
@Rohan27s  please review

### CODE REVIEW

**Comment by Rohan27s** [comment]
mention the issue

**Comment by Amit366** [comment]
Fixes: #167

**Review by Rohan27s — CHANGES_REQUESTED** [review]
Why have you committed two code files in a single PR? Kindly remove one file

**Comment by Amit366** [comment]
@Rohan27s I will add the time complexity and also examples

File: Code/Java/Printing_Shortest_Common_Supersequence.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      
+    // Driver code
+    public static void main(String[] args) 
+    {
+        Scanner sc = new Scanner(System.in);
+        String X = sc.nextLine();
+        String Y = sc.nextLine();;
+        System.out.println(printShortestSuperSeq(X, Y));
+    }
+}
```
Comment (Rohan27s) [inline Code/Java/Printing_Shortest_Common_Supersequence.java:138]: Provide time and space complexity with two test cases

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by Amit366** [comment]
@Rohan27s @tarun26091999 please review

**Review by Rohan27s — APPROVED** [review]

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**